### PR TITLE
Added PetSet support to addon manager.

### PR DESCRIPTION
**Pull request by jszczepkowski**
[Analytics]()

Added PetSet support to addon manager.

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @marekbiskup @fgrzadkowski

**Comment by piosz** [comment]
LGTM from me, @mikedanese could you please take a look?

**Comment by jszczepkowski** [comment]
@mikedanese Can you please comment on this? It seems that you have some knowledge of addon mamanger script.

File: cluster/addons/addon-manager/Makefile [hunk 1]
```diff
@@ -15,26 +15,26 @@
...
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v4
+VERSION=v5
 
 # amd64 and arm has "stable" binaries pushed for v1.2, arm64 and ppc64le hasn't so they have to fetch the latest alpha
 # however, arm64 and ppc64le are very experimental right now, so it's okay
 ifeq ($(ARCH),amd64)
-	KUBECTL_VERSION?=v1.2.4
+	KUBECTL_VERSION?=v1.3.0-beta.2
```
Comment (dims) [inline cluster/addons/addon-manager/Makefile]: Do we need to fix this up again in the 1.3 branch before 1.3 is cut?

Comment (piosz) [inline cluster/addons/addon-manager/Makefile]: This is 'chicken or the egg' problem:(

Comment (idvoretskyi) [inline cluster/addons/addon-manager/Makefile]: @dims @piosz we need to do it in 1.3

**Comment by mikedanese** [comment]
LGTM, did you push all architectures or do you want me to?

**Comment by mikedanese** [comment]
For this to take effect, you need to push all images and bump the version here:

https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/kube-addons/kube-addon-manager.yaml#L13

**Comment by jszczepkowski** [comment]
I didn't push anything yet. I'm wandering why there is no architecture suffix in the yaml file. Should I also push image w/o architecture?

**Comment by mikedanese** [comment]
Pushing amd64 with the makefile will do that for you.

https://github.com/jszczepkowski/kubernetes/blob/51c70f2369437e0c73df63ddb493d4e25fad06b8/cluster/addons/addon-manager/Makefile#L68

**Comment by luxas** [comment]
Seems like @girishkalele pushed it for you now. If no -arch suffix is in the yaml manifest, feel free to change to the binary-arch:version naming scheme in the manifest

**Pull request merged**